gatsby-node.ts

```diff
@@ -1,3 +1,3 @@
-import { createPages } from "./src/gatsby/createPages";
```

Comment:
Why are these commented out?

Reply:
It was intentional. I have  added it back.